tests/system/providers/google/cloud/gcs/example_gcs_upload_download.py

```diff
@@ -57,11 +57,13 @@
     # [END howto_operator_gcs_create_bucket]
 
     # [START howto_operator_local_filesystem_to_gcs]
-    upload_file = LocalFilesystemToGCSOperator(
```

Comment:
Needs to update the comments as well.
`howto_operator_local_filesystem_to_gcs`

Reply:
I think I understood your point. That test was modified without any necessity by mistake. I rolled back those changes. Thank you for noticing.